Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -264,12 +281,29 @@ private bool HandleTryPullObject(ICommonSession? session, EntityCoordinates coor
             return pull.TogglePull(userEntity);
         }
 
-        public async void UserInteraction(IEntity user, EntityCoordinates coordinates, EntityUid clickedUid)
+        /// <summary>
+        ///     Resolves user interactions with objects.
+        /// </summary>
+        /// <remarks>
+        ///     Checks Whether combat mode is enabled and whether the user can actually interact with the given entity. 
+        /// </remarks>
+        /// <param name="altInteract">Whether to use default or alternative interactions (usually as a result of
+        /// alt+clicking). If combat mode is enabled, the alternative action is to perform the default non-combat
+        /// interaction. Having an item in the active hand also disables alternative interactions.</param>
+        public async void UserInteraction(IEntity user, EntityCoordinates coordinates, EntityUid clickedUid, bool altInteract = false )
         {
             if (user.TryGetComponent(out CombatModeComponent? combatMode) && combatMode.IsInCombatMode)
             {
-                DoAttack(user, coordinates, false, clickedUid);
-                return;
+                // alt-interact in combat mode results in normal non-combat interactions
+                // TODO COMBAT Consider using alt-interact for advanced combat? maybe alt-interact disarms?
+                if (altInteract)
+                {
+                    altInteract = false;
+                } else
+                {
+                    DoAttack(user, coordinates, false, clickedUid);
+                    return;
+                }
```

Comment:
This block here means that alt-use in combat mode will use default interactions instead.

This means you can open airlocks while in combat mode, instead of just punching doors.

On the other hand, that functionality is already somewhat covered by the 'E' key, and it also means you cant alt-click to eject a magazine. So maybe this shouldn't be how it works. Should alt-interact just ignore combat mode?